tests/benchmarks/power_consumption/qdec/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.20.0)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(qdec)
+
```

Comment:
if sysbuild is needed then follow https://github.com/nrfconnect/sdk-nrf/blob/main/tests/benchmarks/multicore/idle_with_pwm/CMakeLists.txt